src/main/java/uk/gov/hmcts/reform/lrdapi/util/ValidationUtils.java

```diff
@@ -203,4 +211,56 @@ private static void checkIfStringStartsAndEndsWithComma(String csvIds, String ex
         }
     }
 
+    /**
+     * A util method to check additional filter validation.
+     *
+     * @param requestParam Additional parameters to validate.
+     */
+    public static void validateCourtVenueFilters(CourtVenueRequestParam requestParam) {
+        var hearingLocation = requestParam.getIsHearingLocation();
+        if (hearingLocation != null
```

Comment:
all 3 If statement has similar code. We can write one generic method and pass required params like below

       public static void validateParam(String param, String invalidAdditionalParameter, String filterString) {
        if (param != null
            && !StringUtils.equalsIgnoreCase(param, IS_Y)
            && !StringUtils.equalsIgnoreCase(param, IS_N)) {
            throw new InvalidRequestException(String.format(INVALID_ADDITIONAL_FILTER,
                                                            FILTER_IS_CASE_MANAGEMENT_LOCATION));
}

Reply:
Done !!